### PR TITLE
Add end-to-end unittest of sparse kv column data on PAI MaxCompute

### PR DESCRIPTION
**Pull request by sneaxiy**

### CODE REVIEW

File: go/cmd/sqlflowserver/e2e_common_cases.go [hunk 1]
```diff
@@ -669,78 +685,94 @@ func caseXGBoostSparseKeyValueColumn(t *testing.T) {
...
-	defer executeSQLFunc(dropDBSQL)
-
-	prepareDataSQL := []string{
-		dropDBSQL,
-		fmt.Sprintf(`CREATE DATABASE IF NOT EXISTS %s;`, dbName),
-		fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s.train (c1 VARCHAR(255), label_col FLOAT);`, dbName),
-		fmt.Sprintf(`INSERT INTO %s.train VALUES("1:0.5 3:-1.5 5:6.7", 0.5), ("3:-1.3 10:0.6 2:-3.4", -0.5);`, dbName),
+	var prepareDataSQL []string
+	if !isMaxCompute {
+		prepareDataSQL = []string{
```
Comment (typhoonzero) [inline go/cmd/sqlflowserver/e2e_common_cases.go]: How about put the data preparation process in a function and call it in `sqlflowserver/testing.go:prepareTestData`?

File: go/cmd/sqlflowserver/e2e_common_cases.go [hunk 2]
```diff
@@ -669,78 +685,94 @@ func caseXGBoostSparseKeyValueColumn(t *testing.T) {
...
+	COLUMN SPARSE(c1%s)
+	LABEL label_col
+	INTO %s;
+	`
+
+	trainSQL := fmt.Sprintf(trainSQLTemplate, dbName, trainTable, "", trainedModel)
 	executeSQLFunc(trainSQL)
-	hasModelTableFunc(trainedModel)
+	if !isMaxCompute {
+		hasModelTableFunc(trainedModel)
```
Comment (typhoonzero) [inline go/cmd/sqlflowserver/e2e_common_cases.go:697]: Why? model is saved to OSS only when `SQLFLOW_submitter=pai`, this test should have the model table.

Comment (sneaxiy) [reply go/cmd/sqlflowserver/e2e_common_cases.go:697]: Thanks! Fixed.

File: go/cmd/sqlflowserver/e2e_common_cases.go (hunk 1 above)
Comment (sneaxiy) [reply go/cmd/sqlflowserver/e2e_common_cases.go]: Done.

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**